server/lib/notifications.js

```diff
@@ -161,6 +161,15 @@ export async function notifyAdminsOfCollective(CollectiveId, activity, options =
   return notifySubscribers(adminUsers, activity, options);
 }
 
+/**
+ * Notify all admins of hosted collectives
+ */
+export async function notifyHostedCollectiveAdmins(CollectiveId, activity, options = {}) {
+  const collective = await models.Collective.findByPk(CollectiveId);
+  const adminUsers = collective.getHostedCollectiveAdmins();
+
+  return notifySubscribers(adminUsers, activity, options);
```

Comment:
We're lucky that it worked (we're using `bluebird` in this file, which does some magic with promises) but it's an invalid usage, because `notifySubscribers` check this like `users.length`.

It also generates a warning:
```
Unhandled rejection TypeError: options argument must be an object but it is [object String]
    at map (/home/user/Workspace/Projects/opencollective/opencollective-api/node_modules/bluebird/js/release/map.js:156:35)
    at Promise.map (/home/user/Workspace/Projects/opencollective/opencollective-api/node_modules/bluebird/js/release/map.js:167:12)
    at /home/user/Workspace/Projects/opencollective/opencollective-api/server/lib/notifications.js:105:11
    at Generator.next (<anonymous>)
    at Generator.tryCatcher (/home/user/Workspace/Projects/opencollective/opencollective-api/node_modules/bluebird/js/release/util.js:16:23)
    at PromiseSpawn._promiseFulfilled (/home/user/Workspace/Projects/opencollective/opencollective-api/node_modules/bluebird/js/release/generators.js:97:49)
    at /home/user/Workspace/Projects/opencollective/opencollective-api/node_modules/bluebird/js/release/generators.js:201:15
    at _notifySubscribers (/home/user/Workspace/Projects/opencollective/opencollective-api/server/lib/notifications.js:166:29)
    at notifySubscribers (/home/user/Workspace/Projects/opencollective/opencollective-api/server/lib/notifications.js:130:29)
    at /home/user/Workspace/Projects/opencollective/opencollective-api/server/lib/notifications.js:171:10
    at Generator.next (<anonymous>)
    at Generator.tryCatcher (/home/user/Workspace/Projects/opencollective/opencollective-api/node_modules/bluebird/js/release/util.js:16:23)
    at PromiseSpawn._promiseFulfilled (/home/user/Workspace/Projects/opencollective/opencollective-api/node_modules/bluebird/js/release/generators.js:97:49)
    at Promise._settlePromise (/home/user/Workspace/Projects/opencollective/opencollective-api/node_modules/bluebird/js/release/promise.js:609:26)
    at Promise._settlePromise0 (/home/user/Workspace/Projects/opencollective/opencollective-api/node_modules/bluebird/js/release/promise.js:649:10)
    at Promise._settlePromises (/home/user/Workspace/Projects/opencollective/opencollective-api/node_modules/bluebird/js/release/promise.js:729:18)
    at _drainQueueStep (/home/user/Workspace/Projects/opencollective/opencollective-api/node_modules/bluebird/js/release/async.js:93:12)
    at _drainQueue (/home/user/Workspace/Projects/opencollective/opencollective-api/node_modules/bluebird/js/release/async.js:86:9)
    at Async._drainQueues (/home/user/Workspace/Projects/opencollective/opencollective-api/node_modules/bluebird/js/release/async.js:102:5)
    at Immediate.Async.drainQueues [as _onImmediate] (/home/user/Workspace/Projects/opencollective/opencollective-api/node_modules/bluebird/js/release/async.js:15:14)
    at processImmediate (internal/timers.js:439:21)
    at process.topLevelDomainCallback (domain.js:126:23)
```

We should await: 

```suggestion
  const adminUsers = await collective.getHostedCollectiveAdmins();
  return notifySubscribers(adminUsers, activity, options);
```